src/components/Accordion/Accordion.tsx

```diff
@@ -75,63 +57,43 @@ const AriaAccordionItem: FC<AriaAccordionItemProps> = ({ item, state, header, pa
                         buttonProps.onKeyUp(event);
                     }
                 }}
-                className="tw-w-full tw-px-8 tw-py-6 focus-visible:tw-outline-none"
+                className="tw-w-full focus-visible:tw-outline-none"
             >
-                <FieldsetHeader
-                    {...header}
-                    size={FieldsetHeaderSize.Medium}
-                    active={isOpen}
-                    onClick={undefined}
-                    bold={false}
-                />
+                <HeaderComponent isOpen={isOpen} size={size} {...headerProps} />
             </button>
-
-            <AnimatePresence>
-                {item.props.children && isOpen && (
+            <CollapsibleWrap isOpen={isOpen} preventInitialAnimation={active}>
+                <div {...regionProps} className={merge([padding && "tw-px-8 tw-pb-6"])}>
                     <motion.div
-                        key={item.key}
-                        initial={isActive ? false : "collapsed"}
-                        animate={"open"}
-                        exit={"collapsed"}
-                        variants={{
-                            open: { height: "auto", overflow: "visible" },
-                            collapsed: { height: 0, overflow: "hidden" },
-                        }}
-                        transition={{ type: "tween" }}
-                        data-test-id="accordion-item-content"
+                        initial={{ opacity: 0 }}
+                        animate={{ opacity: 1 }}
+                        exit={{ opacity: 0 }}
+                        transition={{ duration: 0.3 }}
```

Comment:
motion.div is part of collapsibleWrap, is it needed here again?

Reply:
That is a good question. I have removed this and instead added the opacity to the collapsible wrap with the option to turn it off if required, as well as update storybook with this option